content/cockpit-147.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Cockpit 147
+author: Martin Pitt
+date: 2017-07-27 15:00
+tags: cockpit, linux
+slug: cockpit-147
+category: release
+summary: Cockpit with Accounts improvements
+comments: true
+---
+
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release regularly.
+Here are the release notes from version 147.
+
+### Add configuration of expiry to Accounts page
+
+The Accounts page can now display and change the expiration of user accounts
+and their passwords. Correspondingly, the login page now supports changing
+expired passwords.
```

Comment:
The login page has supported this for a while. But triggering it from the Accounts page was missing.